src/components/EpisodeCard/styles.scss

```diff
@@ -0,0 +1,75 @@
+.episode-card {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 16px;
+    gap: 16px;
+    border-radius: 8px;
+    background: $gray;
+    max-width: 368px;
+    width: 100%;
+    box-sizing: border-box;
+
+    &:hover {
+        background: white;
+        box-shadow: -8px 11px 53px $shadow;
```

Comment:
It is ok but you can also define the whole box shadow as variable.

Reply:
Oh i didn't think this. I will do it, thanks :)